deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -39,3 +42,37 @@ def callback_detect_vulnerability_detector_enabled(line):
     match2 = re.match(msg, line)
 
     return match1 is not None and match2 is None
+
+
+def make_callback(pattern):
+    """
+    Creates a callback function from a text pattern.
+
+    It already contains the vulnerability-detector prefix.
+
+    Parameters
+    ----------
+    pattern : str
+        String to match on the log
+
+    >>> callback_bionic_update_started = make_callback("Starting Ubuntu Bionic database update")
+    """
+    pattern = r'\s+'.join(pattern.split())
+    msg = modulesd_prefix + pattern
+    regex = re.compile(msg)
```

Comment:
This block can be done in a single line:
```suggestion
    regex = re.compile(r'{}\s+{}'.format(modulesd_prefix, pattern))
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/689/commits/231750a45f80e84dd90c5506f0b7c3f41b64b775